nix/gecko_env.nix

```diff
@@ -1,6 +1,8 @@
 { releng_pkgs }:
 
 let
+  pkgs = import <nixpkgs> {};
```

Comment:
dont use `<nixpkgs>` since this relies on how your system is configured. use `releng_pkgs.pkgs` instead